antbuild.properties

```diff
@@ -72,6 +72,7 @@ eclipselink.antlr=${eclipselink.plugins}/${antlr}
 eclipselink.mvn.parent=${eclipselink.build.base}/org.eclipse.persistence.parent
 eclipselink.compdeps=${eclipselink.build.base}/compdeps
 eclipselink.oraclelibs=${eclipselink.build.base}/oraclelibs
+eclipselink.sun-misc=${eclipselink.compdeps}/sun.misc
```

Comment:
If I start java with "-Dorg.osgi.framework.system.packages.extra=sun.misc" (osgi 4.3+) then I don't need this bundle. What about just documenting this?

Reply:
I already made those changes. We may keep them in some branch and point documentation on it.